engine/schema/src/main/resources/META-INF/db/schema-41900to41910.sql

```diff
@@ -70,3 +70,5 @@ CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.user_data', 'removed', 'datetime COM
 UPDATE `cloud`.`configuration` SET
     `options` = 'FirstFitRouting,RandomAllocator,TestingAllocator,FirstFitAllocator,RecreateHostAllocator'
 WHERE `name` = 'host.allocators.order';
+
+CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('volumes','last_id', 'bigint(20) unsigned DEFAULT NULL');
```

Comment:
```suggestion

-- Add last_id to the volumes table
CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('volumes','last_id', 'bigint(20) unsigned DEFAULT NULL');
```

Reply:
Looks like maintainers cannot commit/merge suggested changes cc @FelipeM525

Reply:
Also this should ideally be moved to 4.19.1.0->4.19.2.0 sql path

Reply:
@FelipeM525 :point_up_2: